labelbox/schema/project.py

```diff
@@ -63,6 +63,37 @@ class Project(DbObject, Updateable, Deletable):
         benchmarks (Relationship): `ToMany` relationship to Benchmark
         ontology (Relationship): `ToOne` relationship to Ontology
     """
+
+    class MediaType(Enum):
```

Comment:
I don't think these media types are going to be unique to a project. In the future if we were typing a dataset it would be weird to say `client.create_dataset(..., media_type = Project.MediaType.Image)` Let's maybe move this to the data_row.py as a top level enum (top level meaning not nested under a DataRow).

Reply:
The `MediaType` enumeration was moved to its own file (media_type.py) so it could be conveniently used from different classes.